pinot-clients/pinot-jdbc-client/src/main/java/org/apache/pinot/client/controller/response/ControllerTenantBrokerResponse.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.client.controller.response;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectReader;
+import com.google.common.collect.Lists;
+import com.ning.http.client.Response;
+
+import java.io.IOException;
+import java.util.*;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+
+
+public class ControllerTenantBrokerResponse {
+  private JsonNode _brokers;
+
+  private ControllerTenantBrokerResponse() {
+
+  }
+
+  private ControllerTenantBrokerResponse(JsonNode controllerTenantBrokerResponse) {
+    _brokers = controllerTenantBrokerResponse;
+  }
+
+  public static ControllerTenantBrokerResponse fromJson(JsonNode controllerTenantBrokerResponse) {
+    return new ControllerTenantBrokerResponse(controllerTenantBrokerResponse);
+  }
+
+  public static ControllerTenantBrokerResponse empty() {
+    return new ControllerTenantBrokerResponse();
+  }
+
+  public List<String> getBrokers() {
+    List<String> brokerList = new ArrayList<>();
+
+    for (JsonNode broker : _brokers) {
+      String[] brokerPath = broker.textValue().split("_");
```

Comment:
better to get the host name port from instance config instead of the broker name